cartridges/int_algolia/cartridge/templates/default/algolia/headerScripts.isml

```diff
@@ -46,37 +46,37 @@
 
     <!-- Algolia Insights (Click Analytics & Personalization) -->
     <script>
-      var ALGOLIA_INSIGHTS_SRC = "https://cdn.jsdelivr.net/npm/search-insights@1.3.1";
+      var ALGOLIA_INSIGHTS_SRC = "${URLUtils.staticURL('js/lib/algolia/search-insights.min.js')}";
 
       !function(e,a,t,n,s,i,c){e.AlgoliaAnalyticsObject=s,e[s]=e[s]||function(){
       (e[s].queue=e[s].queue||[]).push(arguments)},i=a.createElement(t),c=a.getElementsByTagName(t)[0],
       i.async=1,i.src=n,c.parentNode.insertBefore(i,c)
       }(window,document,"script",ALGOLIA_INSIGHTS_SRC,"aa");
     </script>
-    <script src="${URLUtils.staticURL('/js/algolia/insights-config.js')}"></script>
+    <script src="${URLUtils.staticURL('js/algolia/insights-config.js')}"></script>
 
-    <script src="https://polyfill.io/v3/polyfill.min.js?features=default%2CArray.prototype.find%2CArray.prototype.includes%2CPromise%2CObject.assign%2CObject.entries"></script>
-    <script src="https://cdn.jsdelivr.net/npm/algoliasearch@3/dist/algoliasearchLite.min.js"></script>
-    <script src="https://cdn.jsdelivr.net/npm/hogan.js@1.0.2"></script>
+    <script src="${URLUtils.staticURL('js/lib/algolia/algoliasearchLite.min.js')}"></script>
+    <script src="${URLUtils.staticURL('js/lib/hogan.js')}"></script>
 
     <!-- Algolia Autocomplete.js -->
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@algolia/autocomplete-theme-classic@1" />
+    <link rel="stylesheet" href="${URLUtils.staticURL('css/algolia/autocomplete-theme.min.css')}" />
+
     <script src="https://cdn.jsdelivr.net/npm/@algolia/autocomplete-js@1"></script>
     <script>
       const { autocomplete, getAlgoliaResults } = window['@algolia/autocomplete-js'];
     </script>
     <script type="module">
-        import { html } from 'https://unpkg.com/htm/preact/standalone.module.js'
+        import { html } from '${URLUtils.staticURL("js/lib/preact-standalone.module.js")}'
```

Comment:
Where does it comes from? I don't see it in the dependencies.

Reply:
This particular Preact library is part of the `htm` package, which I also moved to `package.json`, as discussed.